src/conversions/std/slice.rs

```diff
@@ -13,8 +13,8 @@ impl<'a> IntoPy<PyObject> for &'a [u8] {
     }
 }
 
-impl<'a> FromPyObject<'a> for &'a [u8] {
-    fn extract(obj: &'a PyAny) -> PyResult<Self> {
+impl<'py> FromPyObject<'py> for &'py [u8] {
+    fn extract(obj: &'py PyAny) -> PyResult<Self> {
```

Comment:
Similar to that for `Cow`. (The `Cow` one should maybe also move here?)

Reply:
Sadly yes, similar to the above. I'll move the implementation here though 👍